pkg/operation/care/health.go

```diff
@@ -292,6 +293,7 @@ func (h *Health) checkControlPlane(
 
 var versionConstraintGreaterEqual131 *semver.Constraints
 var versionConstraintGreaterEqual132 *semver.Constraints
+var versionConstraintGreaterEqual136 *semver.Constraints
```

Comment:
did I do this right? Max told me, it might be the versions from which the new changes are incorporated, So!

Reply:
Yes, perfect :) Just earliest 1.37 - see above